src/patternfly/components/VerticalNav/styles.scss

```diff
@@ -0,0 +1,134 @@
+  --pf-c-vertical-nav__link--active--BackgroundColor:   var(--pf-global--BackgroundColor--light-200);
+  --pf-c-vertical-nav__link--focus--BackgroundColor:    var(--pf-global--BackgroundColor--light-200);
+  --pf-c-vertical-nav__link--disabled--BackgroundColor: var(--pf-global--BackgroundColor--disabled);
+
+  // Vertical nav link icon
+  --pf-c-vertical-nav__link__icon--MarginRight: var(--pf-global--spacer--xl);
+
+  // Vertical nav link arrow
+  --pf-c-vertical-nav__link__arrow--Top: calc(var(--pf-c-vertical-nav__link--PaddingTop) - .4rem);
+  --pf-c-vertical-nav__link__arrow--Right: var(--pf-global--spacer--xl);
+
+  /* stylelint-enable */
+
+  // List
+  color: var(--pf-c-vertical-nav--Color);
+  background-color: var(--pf-c-vertical-nav--BackgroundColor);
+
+  // List item
+  &__item {
+  }
+
+  // Link
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: baseline;
+    padding-top: var(--pf-c-vertical-nav__link--PaddingTop);
+    padding-right: var(--pf-c-vertical-nav__link--PaddingRight);
+    padding-bottom: var(--pf-c-vertical-nav__link--PaddingBottom);
+    padding-left: var(--pf-c-vertical-nav__link--PaddingLeft);
+    margin-top: var(--pf-c-vertical-nav__link--MarginTop);
+    margin-bottom: var(--pf-c-vertical-nav__link--MarginBottom);
+    color: var(--pf-c-vertical-nav__link--Color);
+    text-decoration: none;
+    background-color: var(--pf-c-vertical-nav__link--BackgroundColor);
+
+    // States
+    &:hover,
+    &:active,
+    &:focus {
+      text-decoration: none;
+    }
+
+    // Modifiers and states
+    &:hover,
+    &.pf-m-hover {
+      color: var(--pf-c-vertical-nav__link--hover--Color);
+      background-color: var(--pf-c-vertical-nav__link--hover--BackgroundColor);
+    }
+
+    &:active,
+    &.pf-m-active {
+      color: var(--pf-c-vertical-nav__link--active--Color);
+      background-color: var(--pf-c-vertical-nav__link--active--BackgroundColor);
+    }
+
+    &:focus,
+    &.pf-m-focus {
+      z-index: 2;
+      color: var(--pf-c-vertical-nav__link--focus--Color);
+      background-color: var(--pf-c-vertical-nav__link--focus--BackgroundColor);
+    }
+
+    &:disabled,
+    &.pf-m-disabled {
+      color: var(--pf-c-vertical-nav__link--disabled--Color);
+      background-color: var(--pf-c-vertical-nav__link--disabled--BackgroundColor);
+    }
+
+    // Link bottom bar - active
+    &.pf-m-active {
+
+      &::after {
+        position: absolute;
+        bottom: 0;
+        left: var(--pf-c-vertical-nav__link--after--active--Left);
+        width: var(--pf-c-vertical-nav__link--after--active--Width);
+        height: var(--pf-c-vertical-nav__link--after--active--Height);
+        content: "";
+        background-color: var(--pf-c-vertical-nav__link--after--active--Color);
+      }
+    }
+
+    // Icon
+    &__icon {
+      margin-right: var(--pf-c-vertical-nav__link__icon--MarginRight);
+      line-height: 1;
```

Comment:
the need to find a way to make this global. Can you please add a comment? I'll create a story to fix the size of the icons somehow.

Reply:
Yep